packages/node/src/integrations/http/SentryHttpInstrumentation.ts

```diff
@@ -353,7 +369,11 @@ function getBreadcrumbData(request: http.ClientRequest): Partial<SanitizedReques
  * we monkey patch `req.on('data')` to intercept the body chunks.
  * This way, we only read the body if the user also consumes the body, ensuring we do not change any behavior in unexpected ways.
  */
-function patchRequestToCaptureBody(req: http.IncomingMessage, isolationScope: Scope): void {
+function patchRequestToCaptureBody(
+  req: http.IncomingMessage,
+  isolationScope: Scope,
+  maxRequestBodySize: 'none' | 'small' | 'medium' | 'always',
```

Comment:
l: We could actually type this to omit `none` as allowed value, and then avoid the further check in here, as we actually guard agains this already before calling this?